sig-architecture/annual-report-2021.md

```diff
@@ -3,80 +3,140 @@
 
 2. What metrics/community health stats does your group care about and/or measure?
 
-   -
-   -
+   - We care about regressions, backports, dependencies across release boundaries
+   - We care about the number of enhancements per release along with their timely graduation
    -
 
 3. Does your [CONTRIBUTING.md] help **new** contributors engage with your group specifically by pointing
    to activities or programs that provide useful context or allow easy participation?
 
-   -
+   - TODO: Riaan
 
 4. If your group has special training, requirements for reviewers/approvers, or processes beyond the general [contributor guide],
    does your [CONTRIBUTING.md] document those to help **existing** contributors grow throughout the [contributor ladder]?
 
-   -
+   - PRR and API review has dedicated docs that can help folks get start. What we need is for SIG leads to send people our way
+       - API review: each SIG can identify 2-3 people to be involved in API reviews - https://github.com/kubernetes/kubernetes/blob/v1.23.0/OWNERS_ALIASES#L451-L452
 
 5. Does the group have contributors from multiple companies/affiliations?
 
-   -
+   - yes
 
 6. Are there ways end users/companies can contribute that they currently are not?
    If one of those ways is more full time support, what would they work on and why?
 
-   -
+   - We need help across all the subprojects that have well defined, ongoing work with an onboarding guide/doc
    -
 
 ## Membership
 
-- Primary slack channel member count:
-- Primary mailing list member count:
-- Primary meeting attendee count (estimated, if needed):
-- Primary meeting participant count (estimated, if needed):
-- Unique reviewers for SIG-owned packages: <!-- in future, this will be generated from OWNERS files referenced from subprojects, expanded with OWNERS_ALIASES files -->
-- Unique approvers for SIG-owned packages: <!-- in future, this will be generated from OWNERS files referenced from subprojects, expanded with OWNERS_ALIASES files -->
+- Primary slack channel member count: #sig-architecture Slack - 2 519
+- Primary mailing list member count: kubernetes-sig-architecture mailing list - 552 members
+- Primary meeting attendee count (estimated, if needed): ~ 10 - 12 
+- Primary meeting participant count (estimated, if needed): ~6 - 8
+- Unique reviewers for SIG-owned packages: 20 <!-- in future, this will be generated from OWNERS files referenced from subprojects, expanded with OWNERS_ALIASES files -->
+- Unique approvers for SIG-owned packages: 13
+<!--
+Used this from the owners [alias file](https://github.com/kubernetes/kubernetes/blob/master/OWNERS_ALIASES)
+Counted Unique approvers and reviewers.
+Did not count approvers again as reviewers. - Riaan
+Enhancements [owners file](https://github.com/kubernetes/enhancements/blob/master/OWNERS_ALIASES)
+- sig-architecture-approvers:
+    - dims
+    - derekwaynecarr
+    - johnbelamaric
+- api-approvers:
```

Comment:
Not so much annual report related, but the fact that we only have 4 API approvers is concerning.

Reply:
yes definitely a bottleneck